tasks/server.coffee

```diff
@@ -28,6 +29,7 @@ module.exports = (grunt) ->
     apiProxyPrefix  = grunt.config.get("server.apiProxy.prefix") || undefined
     apiProxyHost = grunt.config.get("server.apiProxy.host") || "localhost"
     apiProxyChangeOrigin = grunt.config.get("server.apiProxy.changeOrigin")
+    apiProxyUseSsl = process.env.API_USE_SSL || grunt.config.get("server.apiProxy.useSsl") || false
```

Comment:
I added the API_USE_SSL environment variable to be consistent with the API_PORT environment variable. It seems like if you are changing the port, you might need to change whether you are using SSL or not too. However, I don't really see a compelling case for either.

There is a use case where you might change the proxy endpoint (host/port/protocol) from backing dev server to a test server or even production while developing but I would think you would need to either be able to change all of the components or none of them via the environment. Proxy host doesn't have an environment variable so that probably defeats the purpose right there. Personally, I don't use the environment features. If I need to point somewhere else temporarily, I just edit my application.js file and then revert when I'm done.

I'm happy to remove the API_USE_SSL environment variable (and/or update the API host and port to be consistent) and update the pull request if there is consensus.

Let me know what works best for your process and I'll try to follow along.